services/sleeper/docker-compose-meta.yml

```diff
@@ -0,0 +1,75 @@
+          "output_1_label", "description": "A file containing one integer", "type":
+          "data:text/plain", "fileToKeyMap": {"single_number.txt": "output_1"}}, "output_2":
+          {"displayOrder": 2, "label": "output_2_label", "description": "Sleep interval",
+          "type": "integer"}}}'
+        io.simcore.type: '{"type": "computational"}'
+        io.simcore.version: '{"version": "2.0.0"}'
+        org.label-schema.build-date: ${BUILD_DATE}
+        org.label-schema.schema-version: "1.0"
+        org.label-schema.vcs-ref: ${VCS_REF}
+        org.label-schema.vcs-url: https://github.com/ITISFoundation/osparc-services.git
+  sleeper-gpu:
+    build:
+      labels:
+        io.simcore.authors:
+          '{"authors": [{"name": "Andrei Neagu", "email": "neagu@itis.swiss",
+          "affiliation": "IT''IS Foundation"}]}'
+        io.simcore.contact: '{"contact": "neagu@itis.swiss"}'
+        io.simcore.description:
+          '{"description": "A service which awaits for time
+          to pass, used for system validation."}'
+        io.simcore.inputs:
+          '{"inputs": {"input_1": {"displayOrder": 1, "label": "File
+          with int number", "description": "Pick a file containing only one integer",
+          "type": "data:text/plain", "defaultValue": "single_number.txt", "fileToKeyMap":
+          {"single_number.txt": "input_1"}}, "input_2": {"displayOrder": 2, "label":
+          "Sleep interval", "description": "Choose an amount of time to sleep", "type":
+          "integer", "defaultValue": 2}, "input_3": {"displayOrder": 3, "label": "Fail
+          after sleep", "description": "If set to true will cause service to fail
+          after it sleeps", "type": "boolean", "defaultValue": false}}}'
+        io.simcore.integration-version: '{"integration-version": "1.0.0"}'
+        io.simcore.key: '{"key": "simcore/services/comp/sleeper-gpu"}'
+        io.simcore.name: '{"name": "sleeper-gpu"}'
+        io.simcore.outputs:
+          '{"outputs": {"output_1": {"displayOrder": 1, "label":
+          "output_1_label", "description": "A file containing one integer", "type":
+          "data:text/plain", "fileToKeyMap": {"single_number.txt": "output_1"}}, "output_2":
+          {"displayOrder": 2, "label": "output_2_label", "description": "Sleep interval",
+          "type": "integer"}}}'
+        io.simcore.type: '{"type": "computational"}'
+        io.simcore.version: '{"version": "2.0.0"}'
+        org.label-schema.build-date: ${BUILD_DATE}
+        org.label-schema.schema-version: "1.0"
+        org.label-schema.vcs-ref: ${VCS_REF}
+        org.label-schema.vcs-url: https://github.com/ITISFoundation/osparc-services.git
+        simcore.service.settings:
+          '[{ "name": "constraints", "type": "string", "value":
+          ["node.platform.os == linux"]},{"name": "Resources", "type": "Resources", "value":
+          { "Reservations": { "NanoCPUs": 100000000, "MemoryBytes": 536870912,
```

Comment:
really need to reserve CPU and RAM as well for sleeping? probably not

Reply:
No, and thought I'd removed them, this is strange!